LayoutTests/imported/w3c/web-platform-tests/css/css-grid/grid-lanes/tentative/subgrid/row/named-lines-001.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE HTML>
+<!--
+     Any copyright is dedicated to the Public Domain.
+     http://creativecommons.org/publicdomain/zero/1.0/
+-->
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>CSS Grid Lanes Subgrid Test: referencing lines named by subgrid</title>
+  <link rel="author" title="Brandon Stewart" href="mailto:brandonstewart@apple.com">
+  <link rel="help" href="https://drafts.csswg.org/css-grid-3/#subgrids">
+  <link rel="match" href="named-lines-001-ref.html">
+  <style>
+html, body {
+  color: black;
+  background: white;
+  font: 10px/1 monospace;
+  padding: 0;
+  margin: 0;
+}
+
+.grid {
+  display: inline-grid-lanes;
+  grid-lanes-direction: row;
+  grid-template-columns: 40px 60px 80px 100px 120px;
+  border: 1px solid black;
+  margin: 10px;
+  background: lightgrey;
+}
+
+.subgrid {
+  display: grid;
+  grid-template-columns: subgrid [start] [middle] [end];
+  background: lightblue;
+}
+
+.item {
+  background: salmon;
+  border: 1px solid red;
+}
+
+.subitem {
+  background: lightgreen;
+  border: 1px solid green;
+}
+  </style>
+</head>
+<body>
+
+<!-- Subgrid with named lines, items placed using those names -->
+<div class="grid">
+  <div class="item" style="grid-column: 1">A</div>
+  <div class="subgrid" style="grid-column: 2 / 5">
```

Comment:
Make this auto-placed by giving it only a `span 3`. That will check that it works even when it's auto-placed. (Parent line names won't work for an auto-placed subgrid, but the subgrid's own names should work.)